### ISSUE TITLE
"Window" item in dropdown only translated into Japanese & French

### ISSUE THREAD

**Issue by fmaida** [adobe/brackets#10659]
I'm actually testing Brackets 1.2 build 1.2.0.15689 on Mac OS X 10.10.2.
When I change the language in brackets and I switch it to italian, I see that in the dropdown menu the item "Window" is not properly translated into "Finestra". Also, none of the sub-items inside the "Window" item are translated, and they appear in english.

I've also tried to switch to other languages such as French and German, and I noticed that the problem is still present: All the other menu items are properly translated with the exception of "Window".

This is a screenshot I've just taken:

![window_e_menubar](https://cloud.githubusercontent.com/assets/605017/6411059/f26bfc5e-be74-11e4-8fe2-164a7806d7dc.jpg)

Thank you for your time spent developing and debugging brackets

**Comment by nethip** [comment]
@rawat11 Can you check if these translations are missing in JP and Fr? And also please check these in both Brackets 1.1 and Brackets 1.2

**Comment by fmaida** [comment]
I can confirm that the translations are missing for both Japanese and French, and this issue seems to be also present on the official release 1.1

Here's a screenshot from the japanese menu bar as it appears on my computer:

![menubar_e_getting_started_ _brackets_e_download_e_brackets_release_1_1](https://cloud.githubusercontent.com/assets/605017/6412207/2dfa8b30-be81-11e4-978a-052fbed1be18.jpg)

**Comment by nethip** [comment]
@fmaida Thanks for trying out and confirming this! 

@rawat11 Could you confirm if the *Window* menu is supposed to be translated or is the omission intentional?

**Comment by peterflynn** [comment]
@fmaida Those menus will always follow your OS language regardless of any language override you set via the _Debug > Switch Language_ command.  See https://github.com/adobe/brackets/issues/10114#issuecomment-66208410 and the code in PR adobe/brackets-shell#21.

So far we've been considering this not a bug, since the Debug menu item is unofficial (used only for testing), and thus normally _all_ of Brackets follows the OS setting, keeping the menu translations consistent.

**Comment by rawat11** [comment]
@fmaida Yes the menus will follow the OS language. If you do the same in Localized OS, the option would be localized in that particular language.

@nethip As of now its directly picking the OS language. But we can still get it confirmed if we need to provide its translation.

**Comment by rawat11** [comment]
@peterflynn Do you think we need to provide its translation ?

**Comment by peterflynn** [comment]
@rawat11 It's already correctly translated to match the OS language, right?  If so then I would think no additional translations are needed...

**Comment by nethip** [comment]
Closing this as changing the OS language is going to change the locale of the "Window" menu to that of the OS.

**Comment by fmaida** [comment]
@peterflynn My OS is currently set to display messages in Italian, and yet that menu appears in English.

**Comment by peterflynn** [comment]
@fmaida Ah, good point -- only the languages translated by Adobe have localizations for the native menus.  Let's morph this into a bug about that issue...

**Comment by peterflynn** [comment]
The current translations are here:
https://github.com/adobe/brackets-shell/tree/master/appshell/mac/French.lproj/MainMenu.xib
https://github.com/adobe/brackets-shell/tree/master/appshell/mac/Japanese.lproj/MainMenu.xib

We'd need to get community translations matching the other locales that the JS side of Brackets supports.
